components/event-feed-service/pkg/config/service.go

```diff
@@ -38,9 +38,9 @@ type Cereal struct {
 	Address string `mapstructure:"address"`
 }
 
-// Jobs contains the default settings for the jobs scheduler. In many cases
-// these are only applied when the job workflows are created, after which they
-// are managed through gRPC requests and not config.
+// Jobs contains the default settings for the jobs scheduler. This is only applied
+// when the job workflows are created, after which they are managed through the
+// Purge gRPC Configure request, not the config.
```

Comment:
side question for later: would it make sense to have a chef-automate command for Purge?

Reply:
I think it's totally reasonable if we want to do that, but I sorta hope that we could add stuff via the UI to do these things.